### ISSUE TITLE
[BUG] White Page after Login / Internal Error 500

### ISSUE THREAD

**Issue by datWeazel** [Leantime/leantime#1419]
*If you have problems with the installation please use our [community forum](https://community.leantime.io)*

**Describe the bug**
After Updating to the latest Version 2.3.10 I was greeted by the database update prompt. I clicked that, and was redirected to the login after that. After logging in all I get is a white page. 

First I checked the container log of the `leantime` container seeing this: 
```
leantime  | 127.0.0.1 -  12/Feb/2023:17:12:12 +0000 "GET /index.php" 500
```
After that I checked the error log of the `leantime` container (`/var/www/html/logs/error.log`) and saw the following: 
```
[12-Feb-2023 18:12:12 Europe/Stockholm] PDOException: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ') then 1 else 0 end) as sum_open_todos,
                        SUM(case when (s' at line 4 in /var/www/html/app/domain/reports/repositories/class.reports.php:91
Stack trace:
#0 /var/www/html/app/domain/reports/repositories/class.reports.php(91): PDOStatement->execute()
#1 /var/www/html/app/domain/reports/services/class.reports.php(73): leantime\domain\repositories\reports->runTicketReport('4', '')
#2 /var/www/html/app/domain/dashboard/controllers/class.home.php(88): leantime\domain\services\reports->getRealtimeReport('4', '')
#3 /var/www/html/app/core/class.controller.php(61): leantime\domain\controllers\home->get(Array)
#4 /var/www/html/app/core/class.controller.php(33): leantime\core\controller->executeActions('get', Array)
#5 /var/www/html/app/core/class.frontcontroller.php(176): leantime\core\controller->__construct('get', Array)
#6 /var/www/html/app/core/class.frontcontroller.php(98): leantime\core\frontcontroller::executeAction('dashboard.home', Array, 200)
#7 /var/www/html/app/core/class.application.php(119): leantime\core\frontcontroller::dispatch()
#8 /var/www/html/public/index.php(26): leantime\core\application->start()
#9 {main}
```

**Leantime Version**
2.3.10 via Docker

**Server**
**PHP / MySQL Version**
Using the Docker Image

**Comment by marcelfolaron** [comment]
Thank you for your submission. I've noticed that only once before and it had something to do with the status set up. 
Do you have access to the database? If so, could you send me the content of:
table: zp_settings
key: projectsettings.{{PROJECTID}}.ticketlabels

It should be a serialized string.

**Comment by marcelfolaron** [comment]
NVM I think I found it. Is it possible that you have no status that is reported as "NEW" ?

**Comment by marcelfolaron** [comment]
Fixed in 2.3.11
